packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -4230,66 +4228,86 @@ export class ContainerRuntime
 		const { proposalHandle, ackHandle, summaryRefSeq, summaryLogger } = options;
 		// proposalHandle is always passed from RunningSummarizer.
 		assert(proposalHandle !== undefined, 0x766 /* proposalHandle should be available */);
-		const readAndParseBlob = async <T>(id: string) => readAndParse<T>(this.storage, id);
 		const result = await this.summarizerNode.refreshLatestSummary(
 			proposalHandle,
 			summaryRefSeq,
 		);
 
 		/**
-		 * When refreshing a summary ack, this check indicates a new ack of a summary that is newer than the
-		 * current summary that is tracked, but this summarizer runtime did not produce/track that summary. Thus
-		 * it needs to refresh its state. Today refresh is done by fetching the latest snapshot to update the cache
-		 * and then close as the current main client is likely to be re-elected as the parent summarizer again.
+		 * If the snapshot corresponding to the ack is not tracked by this client, it was submitted by another client.
+		 * Take action as per the following scenarios:
+		 * - If that snapshot is older than the one tracked by this client, ignore the ack because only the latest
+		 * snapshot is tracked.
+		 * - If that snapshot is newer, attempt to fetch the latest snapshot and do one of the following:
+		 * - If the fetched snapshot is same or newer than the one for which ack was received, close this client. The
```

Comment:
nit: I'd number these sub-bullets instead of having `-` to make it more readable

Reply:
Improved the indentation. It wasn't allowing it until I disabled it.